### PR TITLE
RANCHER-1201 Add dcb, circulation modules and edge-dcb

### PR DESCRIPTION
**Pull request by DmytrMoroz**

### CODE REVIEW

**Review by renatsafiulin — APPROVED** [review]

File: resources/edge/config.yaml [hunk 1]
```diff
@@ -24,6 +24,14 @@ edge-orders:
 edge-ncip:
   permissions:
   - ncip.all
+edge-dcb:
```
Comment (eldiiar-duishenaliev) [inline resources/edge/config.yaml:27]: change it to edge-dcb: {}

Comment (eldiiar-duishenaliev) [reply resources/edge/config.yaml:27]: Similar to https://github.com/folio-org/pipelines-shared-library/blob/RANCHER-741-Jenkins-Enhancements/resources/edge/config.yaml#L60

**Review by OHaimanov — CHANGES_REQUESTED** [review]
Please resolve requested changes

File: resources/edge/config.yaml (hunk 1 above)
Comment (DmytrMoroz) [reply resources/edge/config.yaml]: Done

**Pull request merged**